src/pages/Quiz.tsx

```diff
@@ -18,25 +20,26 @@ interface Props extends RouteComponentProps {
 const Quiz: React.FC<Props> = ({ quiz }) => {
   return (
     <IonPage>
-      <IonContent fullscreen class="ion-padding">
+        
+        <IonImg className="home-lines" src={Try}></IonImg>
+        <IonImg className="logo" src={Logo}></IonImg>
+        <IonButton size="large" className="welcome-button" routerLink="/quiz/question">
+                WELCOME
+              </IonButton>
+     
+       
         <IonGrid className="welcome-content">
           <IonRow>
             <IonCol>
-              <IonImg src="/assets/logo.png" />
               <h1 className="subtitle">{quiz && quiz.name}</h1>

```

Comment:
This is the text which should be under the logo - right now it is hidden on most screen widths but will break the site layout on weird ratios/small widths.